main.go

```diff
@@ -10,8 +10,7 @@ import (
 func main() {
 	plugin.Serve(&plugin.ServeOpts{
 		RuleSet: &tflint.BuiltinRuleSet{
-			Name:    "kafka-config",
-			Version: "0.1.0",
```

Comment:
for the future: you can set the version from `go-releaser`, minimal effort example https://github.com/utilitywarehouse/release-tools/blob/2d644fc7749798abeb4f5862c21dbe463f9987bb/cmd/uw-releaser/main.go#L23

Reply:
Nice. Included it in here too.